examples/lc709203f/default/main/main.c

```diff
@@ -0,0 +1,86 @@
+#include <stdio.h>
+#include <string.h>
+
+#include "freertos/FreeRTOS.h"
+#include "freertos/task.h"
+
+#include "esp_err.h"
+#include "esp_log.h"
+
+#include "driver/gpio.h"
```

Comment:
use `#include <foo.h>`, instead of `#include "foo.h"`, when the header is not one in the component.

Reply:
for ```include <xxx.h>```, I follow 'esp-idf' design guides. 

[ESP-IDF desing guides](https://docs.espressif.com/projects/esp-idf/en/v4.4.1/esp32/contribute/style-guide.html#include-statements)
> Use angle brackets for C standard library headers and other POSIX headers (#include <stdio.h>).
>
>Use double quotes for all other headers (#include "esp_log.h").

I have no problem to change double quotes but, maybe, design guides should be the same.